kinto/core/initialization.py

```diff
@@ -335,14 +335,15 @@ def on_new_request(event):
                 errno=errors.ERRORS.INVALID_PARAMETERS,
                 message='Invalid URL path.')
 
+        qs = dict(errors.request_GET(request))
         request.log_context(agent=request.headers.get('User-Agent'),
                             path=request_path,
                             method=request.method,
-                            querystring=dict(errors.request_GET(request)),
+                            querystring=(qs if len(qs) else None),
```

Comment:
You could shorten this to `qs or None`. But why not just build a dict of parameters we want to include and only add `querystring` if there's something here, similar to the way you do in the error view?